### PR TITLE
LPS-147906 Allow nestedFieldsDepth parameter to be added at query level to limit the depth of nested fields

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://issues.liferay.com/browse/LPS-147906

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java [hunk 1]
```diff
@@ -82,36 +84,26 @@ public ObjectEntry toDTO(
...
+				dtoConverterContext, objectEntry,
+				Math.min(1, PropsValues.OBJECT_NESTED_FIELDS_MAX_QUERY_DEPTH));
+		}
+
+		MultivaluedMap<String, String> queryParameters =
+			uriInfo.getQueryParameters();
+
+		return _toDTO(
+			dtoConverterContext, objectEntry,
+			Math.min(
```
Comment (rodrigopaulino) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java:103]: Nice!

**Review by rodrigopaulino — APPROVED** [review]
👏🏻

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**